tmp/lanelet2_extension/docs/lanelet2_format_extension.md

```diff
@@ -202,6 +202,28 @@ The following illustrates how light_bulbs are registered to traffic_light regula
 </relation>
 ```
 
+### Crosswalk
+
```

Comment:
It might be better to add why we are adding this regulatory element in Autoware while original Lanelet2 doesn't require regulatory element.

Example:
```suggestion
Original Lanelet2 format only requires `subtype=crosswalk` tag to be specified in the corresponding lanelet. However, Autoware requires a regulatory element to be defined on top of that in order to:
- explicitly define the relevant driving lanes even in 3D environment
- optionally define stop lines associated with the crosswalk
- enable accurate definition of complex polygons for crosswalk

For the details, refer to this [GitHub discussion](https://github.com/orgs/autowarefoundation/discussions/3036).
```

Reply:
Got it. Thanks for your suggestion!